ChangeLog.d/Driver_wrapper_codegen_1.0.txt

```diff
@@ -0,0 +1,6 @@
+Features
+   * Add support for driver wrapper codegen 1.0. This autogenerates
+     the file library/psa_crypto_driver_wrappers.c. Rev 1.0 does not
+     support any additional templating. For further info on how to
+     patch into the psa_crypto_driver_wrappers.c refer
+     docs/proposed/psa-driver-wrappers-codegen-migration-guide.md
```

Comment:
Please keep in mind that the target audience of this file is everyone who uses the library. Most readers will have never heard of “driver wrapper codegen 1.0”. The changelog entry should explain what changed. It should generally not mention things that haven't happened yet, although here it does make sense to explain the reason for the change (the file is generated now, because it will become variable later). Suggestion:

> The file library/psa_crypto_driver_wrappers.c is now generated from a template. In the future, the generation will support driver descriptions. For the time being, to customize this file, see docs/proposed/psa-driver-wrappers-codegen-migration-guide.md

Since there isn't yet any useful feature from the templating, this should go under “Changes” (to the build process) rather than “Features”.

We should revisit this changelog entry when 3.2 is released: depending on when the release happens, it may need to be consolidated with announcements of features that the generator does support by that point.